tools/generate_plugindata/main_test.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGeneratePluginData(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
```

Comment:
These two tests take 0.002s on my system, I don't think we should skip them unless there was some concern?

Reply:
The issue I ran into was in the Windows CI test job, because the test creates/removes files it failed with: 
`rename test.go test.go.tmp: The process cannot access the file because it is being used by another process.`
I couldn't quickly find a good way to get around this, so just muting them seemed like the best idea seeing they are very integration tests like.